iceoryx_hoofs/test/moduletests/test_lockfree_queue_buffer.cpp

```diff
@@ -32,22 +32,11 @@ class LockFreeQueueBufferTest : public ::testing::Test
     using Buffer = T;
 
   protected:
-    LockFreeQueueBufferTest()
-    {
-    }
-
-    ~LockFreeQueueBufferTest()
-    {
-    }
+    LockFreeQueueBufferTest() = default;
 
-    void SetUp()
-    {
-    }
-
-    void TearDown()
-    {
-    }
+    ~LockFreeQueueBufferTest() override = default;
 
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-type-member-init) false positive
```

Comment:
@elBoberido This suppression also does not work for me and again I think it is a false positive. The buffer is default initialized in the default ctor, there should be no problem (at least the way I understand the warning).

The problem is that the rule may find valid problems with uninitialized members as well, but if it catches these kind of false positives it is useless and should be disabled I think.